Description: Omit bogus placeholder glyphs from built fonts
 We mustn't include glyphs for unallocated codepoints, since the whole
 idea is that those get allocated later, which leads to useless
 rendering (the very thing this package was intended to prevent).
 Thus, we drop blanks.hex.
 .
 It also doesn't make sense to fill the private use area with
 placeholders: someone might actually want to *use* it.
 Thus, we drop rc-priv.hex.

Author: Samuel Bronson <naesten@gmail.com>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Last-Update: 2013-01-11

--- unifont-5.1.20080914.orig/font/Makefile
+++ unifont-5.1.20080914/font/Makefile
@@ -44,15 +44,15 @@ HEXDIR = hexsrc
 # Wen Quan Yi CJK ideographs.  This version provides complete coverage
 # of the Unicode 5.1 Basic Multilingual Plane.
 #
-UNIFILES = $(HEXDIR)/blanks.hex $(HEXDIR)/rc-base.hex $(HEXDIR)/wqy-cjk.hex \
-	$(HEXDIR)/rc-hangul.hex $(HEXDIR)/rc-priv.hex
+UNIFILES = $(HEXDIR)/rc-base.hex $(HEXDIR)/wqy-cjk.hex \
+	$(HEXDIR)/rc-hangul.hex
 #
 # These are the files for building GNU Unifont with Roman Czyborra's
 # original Japanese public domain CJK ideographs.  This version is not
 # complete, as it is missing thousands of CJK ideographs.
 #
-UNIORIGFILES = $(HEXDIR)/blanks.hex $(HEXDIR)/rc-base.hex \
-	$(HEXDIR)/rc-cjk.hex $(HEXDIR)/rc-hangul.hex $(HEXDIR)/rc-priv.hex
+UNIORIGFILES = $(HEXDIR)/rc-base.hex \
+	$(HEXDIR)/rc-cjk.hex $(HEXDIR)/rc-hangul.hex
 #
 # Location of  TTF source directory, where TTF font is built.
 #
